packages/components/src/styles/components/charts.scss

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// CHARTS
+//
+
+// > DONUT
+
+.hds-charts-donut {
+  // outline: 1px dotted red; // for testing - see boundaries of chart container
+
+  // increase stroke definition on slices (especially for resized/shrunken charts)
+  .slice {
+    stroke: #fff;
+    stroke-width: 1px;
+  }
```

Comment:
I added this to improve the crispness of the thin white border on resized components, just as a test. (Not sure if we would actually want to recommend)